packages/dev/core/src/Engines/WebGL/webGLHardwareTexture.ts

```diff
@@ -52,6 +52,10 @@ export class WebGLHardwareTexture implements HardwareTextureWrapper {
         }
     }
 
+    public get mSAARenderBuffer() {

```

Comment:
Is it not weird to have a shortcut for index 0 only ?

Reply:
for my use case, I only needed index 0 as it is the main one. I'm not sure in what case we need several MSAA buffers, texture arrays ?

Reply:
yup I think multi RTT.

So opening it for 0 only sounds pretty specific ?

Reply:
multi RTT still have N webGLHardwareTextures, each one having their own mSAARenderBuffer, I think the use case is different for this array.
Anyway your point is still valid, it's specific for whenever you want to access the main one, which is always index 0